src/components/FiltersDrawer/index.tsx

```diff
@@ -1,24 +1,32 @@
 import { useState } from "react";
 import styled from "@emotion/styled";
 import Badge, { Variant } from "@leafygreen-ui/badge";
+import IconButton from "@leafygreen-ui/icon-button";
 import { palette } from "@leafygreen-ui/palette";
 import { SideNav, SideNavGroup } from "@leafygreen-ui/side-nav";
-import { Body } from "@leafygreen-ui/typography";
+import { Body, Overline } from "@leafygreen-ui/typography";
 import Cookie from "js-cookie";
 import Icon from "components/Icon";
 import { HAS_OPENED_DRAWER } from "constants/cookies";
 import { QueryParams } from "constants/queryParams";
 import { size, zIndex } from "constants/tokens";
 import { useQueryParam } from "hooks/useQueryParam";
+import { ExpandedLines } from "types/logs";
 import Filter from "./Filter";
 
 const { green, gray } = palette;
 
 interface FiltersDrawerProps {
   ["data-cy"]?: string;
+  expandedLines: ExpandedLines;
+  collapseLines: (idx: number) => void;
 }
 
-const FiltersDrawer: React.FC<FiltersDrawerProps> = ({ "data-cy": dataCy }) => {
+const FiltersDrawer: React.FC<FiltersDrawerProps> = ({
```

Comment:
I think it's okay because Parsley is on React 18 ^^ Spruce is on 17, so it needs to use `VFC`

Reply:
Yup the change was made in the update to react 18. `React.FC` no longer includes children by default and is the recommended way to go. https://github.com/DefinitelyTyped/DefinitelyTyped/commit/55dc209ceb6dbcd59c4c68cc8dfb77faadd9de12#diff-32cfd8cb197872bcba371f5018185d2e75fa540b52cda2dd7d8ac12dcc021299L500

We used React.VFC on Spruce since the typings for React 17 explicitly typed children which is error prone.